.github/workflows/deploy-to-azure-app-service.yml

```
@@ -80,7 +80,8 @@ on:
       slot-name:
         required: true
         type: string
-        description: What you see at the top of the blade on the Azure Portal, when you open the slot, before the app
+        description:
+          What you see at the top of the blade on the Azure Portal, when you open the slot, before the app
```

While this doesn't matter much in practice right now, this adds a line break to the value of `description`, as opposed to the value being a single-line string (what we'd need if this were displayed in some UI) and only having line for the editing experience..

```suggestion
        description: >
          What you see at the top of the blade on the Azure Portal, when you open the slot, before the app
```

---

I agree! I was experimenting with some settings and modified this one with line wrapping set to "preserve", which caused it not to move the beginning of the text to first line even without `>`.
Auto-wrapping can be improved even further by changing a setting in VS Code, I'll try that and add it to the instructions.